index.html

```diff
@@ -14,21 +22,22 @@
     <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&display=swap" rel="stylesheet">
   </head>
   <body id="body">
-    <div id="overlay"></div>
-    <header id="header" class="container">
-      <a href="#" class="logo">Welcome</a>
-      <img src="./imges/Union.svg" id="menu-btn" alt="menu" ></a></li>
+    <header>
+      <div id="header" class="container">
+        <a href="#" class="logo">Welcome</a>
+        <button type="button" class="menu-btn">
+          <img src="./imges/Union.svg" alt="menu">
+        </button>   
+        <nav class="header-nav-mobile hidden">
+            <img src="./imges/Icon-Cancel.svg" alt="cancel icon" class="cancel-cross">
+          <ul>
+            <li><a href="#myRecentWork" class="menu-link">Portfolio</a></li>
+            <li><a href="#about-me" class="menu-link">About</a></li>
+            <li><a href="#section-contact" class="menu-link">Contact</a></li>
+          </ul>
```

Comment:
Please make sure the above requirements are met ⬇️ 
- [ ]  When the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed

- [ ] When the user clicks (or taps) any option of the mobile menu, the mobile menu disappears

- Kindly check that you have given  `id` names to the section you want displayed on the mobile menu, I see you have used class names instead in [portfolio section](https://github.com/cilfonegabriel/portfolio/blob/8ce4dfce053f1bd15f5b28833333b4224a735cf3/index.html#L73) and [about me section](https://github.com/cilfonegabriel/portfolio/blob/8ce4dfce053f1bd15f5b28833333b4224a735cf3/index.html#L150)